pkg/task/handler/handler.go

```diff
@@ -44,62 +43,71 @@ type groupAlterMessageHandler struct {
 func NewGroupAlterMessageHandler() MessageHandler {
 	return &groupAlterMessageHandler{
 		groupService:                      service.NewGroupService(),
-		groupAlterEventService:            service.NewGroupAlterEventService(),
+		subjectActionAlterMessageService:  service.NewSubjectActionAlterMessageService(),
 		subjectActionGroupResourceService: service.NewSubjectActionGroupResourceService(),
 		subjectActionExpressionService:    service.NewSubjectActionExpressionService(),
 		locker:                            locker.NewDistributedSubjectActionLocker(),
 	}
 }
 
 // Handle ...
-func (h *groupAlterMessageHandler) Handle(message string) (err error) {
+func (h *groupAlterMessageHandler) Handle(uuid string) (err error) {
 	errorWrapf := errorx.NewLayerFunctionErrorWrapf(handlerLayer, "handleEvent")
 
-	pk, err := strconv.ParseInt(message, 10, 64)
+	message, err := h.subjectActionAlterMessageService.Get(uuid)
 	if err != nil {
-		err = errorWrapf(err, "parse message to pk fail, message=`%s`", message)
+		err = errorWrapf(err, "subjectActionAlterMessageService.Get event fail, uuid=`%s`", uuid)
 		return err
 	}
 
-	event, err := h.groupAlterEventService.Get(pk)
+	// update message status to processing
+	err = h.subjectActionAlterMessageService.BulkUpdateStatus(
+		[]string{uuid},
+		types.SubjectActionAlterMessageStatusProcessed,
```

Comment:
SubjectActionAlterMessageStatusProcessed to SubjectActionAlterMessageStatusProcessing